packages/perseus-score/src/validation.types.ts

```diff
@@ -225,6 +225,11 @@ export type PerseusNumericInputUserInput = {
     currentValue: string;
 };
 
+export type PerseusFreeResponseUserInput = {
+    currentValue: string;
+    question: string;
```

Comment:
Why include the question here?

Reply:
Good question. That was an initial attempt to make it easy to extract the question from the widget, but we've realized it's not needed, so I've removed it.